stringResources/en-US/resources.resjson

```diff
@@ -9,6 +9,16 @@
 	"Visual_General": "General",
 	"Visual_GroupTasks": "Group Tasks",
 	"Visual_ScrollToCurrentTime": "Scroll to current time",
+	"Visual_DaysOff": "Days off",
+    "Visual_Day" : "Day",
+    "Visual_Day_Sunday": "Sunday",
+    "Visual_Day_Monday": "Monday",
+    "Visual_Day_Tuesday": "Tuesday",
+    "Visual_Day_Wednesday": "Wednesday",
+    "Visual_Day_Thursday": "Thursday",
+    "Visual_Day_Friday": "Friday",
+    "Visual_Day_Saturday": "Saturday",
+    "Visual_FirstDayOfWeek": ""First day of week,
```

Comment:
Looks like indent isn't correct. Please format this file properly.

Reply:
Done